src/figuresClasses.ts

```diff
@@ -1,19 +1,93 @@
+type Shape = 'triangle'| 'circle'| 'rectangle';
+type Color = 'red' | 'green' | 'blue';
+
 export interface Figure {
+  shape: Shape;
+  color: Color;
+  getArea(): number;
+}
 
+function roundedArea(number: number): number {
+  return Math.floor(number * 100) / 100;
 }
 
-export class Triangle {
+export class Triangle implements Figure {
+  shape: Shape = 'triangle';
```

Comment:
will be better to create enum for corresponding values, because for now you can make error while specifying the string